lib/archethic/networking.ex

```diff
@@ -7,6 +7,21 @@ defmodule Archethic.Networking do
   alias __MODULE__.PortForwarding
 
   @ip_validate_regex ~r/(^0\.)|(^127\.)|(^10\.)|(^172\.1[6-9]\.)|(^172\.2[0-9]\.)|(^172\.3[0-1]\.)|(^192\.168\.)/
+
+  def validate_ip(ip, ip_validation? \\ should_validate_node_ip?())
+
+  def validate_ip(ip, true) do
+    case valid_ip?(ip) do
+      true ->
+        :ok
+
+      _ ->
+        {:error, :invalid_ip}
+    end
```

Comment:
```suggestion
    if !valid_ip?(ip), do: {:error, :invalid_ip}
```
Any how if will send you the `:ok` once it is fall for else case

Reply:
If you prefer multiline, we could do like this :
```elixir
if valid_ip?(ip) do
   :ok
else
   {:error, :invalid_ip}
end

Reply:
thanks